packages/lib/src/utils/color.ts

```diff
@@ -37,5 +41,8 @@ export default (hexColor: string) => {
   s = +(s * 100).toFixed(1);
   l = +(l * 100).toFixed(1);
 
-  return [h, `${s}%`, `${l}%`];
+  return [h, s, l];
 };
+
+export const hslArrayToCSS = (hslArray: HSLArray): string =>
```

Comment:
I'm a fan of exporting named functions instead of anonymous functions in named constants :) I personally find it easier to read, but it's a preference

Reply:
Anonymous functions are very much a way of the "functional programmers" to which side I'm more drawn.

In my head the `function` keyword is redundant and leads to more code:
Seeing "const something = `() => {}` the last part indicates that it is a function, as much as `function` spelled out does.

But hey, I'm also the one who rather type:
```
truthy && runPieceOfCode();
```
over
```
if (truthy) {
  runPieceOfCode();
}
```

(which I refrain from when working in a team 😅)